### PR TITLE
[stable/openvpn] Adds nodeSelector

### LINKED ISSUE #9074: [stable/openvpn] Add nodeSelector
**Is this a request for help?**:
No
---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):

Add nodeSelector

**Version of Helm and Kubernetes**:
```
➜  kubernetes_templates git:(explicit-node-scheduling) helm version
Client: &version.Version{SemVer:"v2.11.0", GitCommit:"2e55dbe1fdb5fdb96b75ff144a339489417b146b", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.11.0", GitCommit:"2e55dbe1fdb5fdb96b75ff144a339489417b146b", GitTreeState:"clean"}
➜  kubernetes_templates git:(explicit-node-scheduling) kubectl version
Client Version: version.Info{Major:"1", Minor:"12", GitVersion:"v1.12.0", GitCommit:"0ed33881dc4355495f623c6f22e7dd0b7632b7c0", GitTreeState:"clean", BuildDate:"2018-09-28T15:18:13Z", GoVersion:"go1.11", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"10+", GitVersion:"v1.10.7-gke.6", GitCommit:"06898a4d0f2b96f82b43d9e59fa2825bd3d616a2", GitTreeState:"clean", BuildDate:"2018-10-02T17:32:01Z", GoVersion:"go1.9.3b4", Compiler:"gc", Platform:"linux/amd64"}
```
**Which chart**:
stable/openvpn

**What happened**:

**What you expected to happen**:

**How to reproduce it** (as minimally and precisely as possible):

**Anything else we need to know**:

### PR DESCRIPTION
**Pull request by dolftax**
#### What this PR does / why we need it:

Adds `nodeSelector` to schedule pods in specific nodes.

#### Which issue this PR fixes
  - fixes #9074

#### Special notes for your reviewer:
nil

#### Checklist
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by sameersbn** [comment]
Hi @jaipradeesh, could you please rebase the PR?

/ok-to-test

**Comment by jaipradeesh** [comment]
@sameersbn Sorry for the delay. Rebased.

File: stable/openvpn/Chart.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ description: A Helm chart to install an openvpn server inside a kubernetes clust
   generation is also part of the deployment, and this chart will generate client keys
   as needed.
 name: openvpn
-version: 3.11.0
+version: 3.11.1
```
Comment (cpanato) [inline stable/openvpn/Chart.yaml]: adding new functionality 
```suggestion
version: 3.12.0
```

**Comment by cpanato** [comment]
@jaipradeesh please sign the dco

**Comment by jaipradeesh** [comment]
@cpanato Poke

**Comment by cpanato** [comment]
@jaipradeesh poke back 😄

**Review by cpanato — APPROVED** [review]

**Pull request merged**